File: lab/javascript/pares/filtrar_pares.js
const prompt = require('prompt-sync')();

function filtrar_pares(numero){
    let entrada = prompt();
    let lista = entrada.split(" ").map(Number);

    if (lista.length !== numero){
        throw new Error(
            "Tamanho da lista precisa compadecer ao limite esperado"
        );
    }

    lista.forEach(element => {
        if (element % 2 == 0){
            process.stdout.write(element + " ");;
        }
    });
    console.log("");
}


let numero = Number(prompt())
filtrar_pares(numero)

File: lab/python/pares/filtrar_pares.py
def filtrar_pares(numero):
    lista = list(map(int, input().split()))

    if len(lista) != numero:
        raise IndexError(
            "Tamanho da lista precisa compadecer ao limite esperado"
        )

    for numeros in lista:
        if numeros % 2 == 0:
            print(numeros, end=" ")
    print()


numero = int(input())
filtrar_pares(numero)
